QueryBuilder.php

```diff
@@ -505,7 +505,8 @@ public function buildFrom($indexes, &$params)
                 }
                 $indexes[$i] = "$index " . $this->db->quoteIndexName($i);
             } elseif (strpos($index, '(') === false) {
-                if (preg_match('/^(.*?)(?i:\s+as|)\s+([^ ]+)$/', $index, $matches)) { // with alias
+                if (preg_match('/^(.*?)(?i:\s+as|)\s+([^ ]+)$/', $index, $matches)) {
+                    // with alias
```

Comment:
It's better to be reverted. In the changed variant it's not as clear which line this comment belongs to.